Error in OReportModule caused by OReportSkeletonComponent

ERROR in Unexpected value 'undefined' declared by the module 'OReportModule in C:/git/sefact-vigo/sefact-front/src/main/ngx/node_modules/ontimize-web-ngx-report/ontimize-web-ngx-report.d.ts'
Cannot determine the module for class OReportSkeletonComponent in C:/git/sefact-vigo/sefact-front/src/main/ngx/node_modules/ontimize-web-ngx-report/ontimize-web-ngx-report.d.ts! Add OReportSkeletonComponent
to the NgModule to fix it.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sefact-vigo@8.4.1 production: `ng build --configuration=production --base-href=/app/`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the sefact-vigo@8.4.1 production script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

--- a/projects/ontimize-web-ngx-report/src/lib/config/o-components.ts
+++ b/projects/ontimize-web-ngx-report/src/lib/config/o-components.ts
@@ -14,7 +14,8 @@ import { SavePreferencesDialogComponent } from '../components/report-on-demand/s
 import { ApplyConfigurationDialogComponent } from '../components/report-on-demand/apply-configuration/apply-configuration-dialog.component';
 import { OReportsTranslatePipe } from '../util/o-reports-translate.pipe';
 import { NgxSkeletonLoaderModule } from 'ngx-skeleton-loader';
-import { OReportSkeletonComponent } from '../components';
+import { OReportSkeletonComponent } from '../components/report/o-report-skeleton/o-report-skeleton.component';
+
 
 
 export const OREPORT_DECLARATION_MODULES: any = [

